src/js/jsx/sections/layers/LayersPanel.jsx

```diff
@@ -615,6 +646,14 @@ define(function (require, exports, module) {
                         visible={this.props.visible}
                         disabled={this.props.disabled}
                         onDoubleClick={this.props.onVisibilityToggle}>
+                        <SearchBar
+                            ref="searchBar"
+                            searchID={this.state.searchID}
+                            startFocused={false}
+                            dismissDialog={this._closeSearchBar}
+                            executeOption={this._handleOption}
+                            placeholder={""}
```

Comment:
This should just be omitted and the `SearchBar` should make "" the default prop.